tests/benchmarks/model_benchmarks/test_pytorch_gpt2.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
@guoshzhao After the discussion, we should keep the code coverage on patch is higher than 80%

Reply:
Done. Currently `codecov/patch — 94.68% of diff hit (target 80.00%)`